src/euphorie/client/tests/test_model.py

```diff
@@ -495,7 +495,7 @@ def test_get_account_filter(self):
         )
         self.assertEqual(
             str(session.get_account_filter([account, "2"])),
-            "session.account_id IN (:account_id_1, :account_id_2)",
+            "session.account_id IN (__[POSTCOMPILE_account_id_1])",
```

Comment:
This looks scary. What is that `POSTCOMPILE` prefix and what does the `__`  do?

Reply:
They are just strings like they were before.
How can the new code be scarier than the old code?